src/api/app/assets/stylesheets/webui/responsive_ux.scss

```diff
@@ -6,16 +12,25 @@ body.responsive-ux {
   @import 'responsive_ux/build-result';
   @import 'responsive_ux/sponsors';
 
-  .build-results .sticky-top { top: 57px; }
+  .build-results .sticky-top { top: $top-bar-height; }
 }
 
 // FIXME: move to somewhere else?
 
 @include media-breakpoint-up(md) {
+
   body.responsive-ux {
     // START LAYOUT
+    .flash-and-announcement.sticky-top { top: $md-top-bar-height + 16px; }
```

Comment:
@saraycp to me the 16px extra space is a bit much, maybe going with 8px? :)

Reply:
You are right, and I think even 4px is enough. Updated!